controllers/getControllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopicsFromDb } = require("../models/getModels");
+
+exports.getTopics = (req, res) => {
+  fetchTopicsFromDb().then((topics) => {
+    res.status(200).send({ topics });
+  });

```

Comment:
You will probably want some kind of catch block here to achieve error handling

Reply:
Done :)